ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -123,38 +134,42 @@ public class BottomSheetController: UIViewController {
     }
 
     /// Height of the top portion of the content view that should be visible when the bottom sheet is collapsed.
-    @objc open var collapsedContentHeight: CGFloat = Constants.defaultCollapsedContentHeight {
+    ///
+    /// When set to 0, `headerContentHeight` will be used.
+    @objc open var collapsedContentHeight: CGFloat = 0 {
         didSet {
-            if isViewLoaded {
-                if currentExpansionState == .collapsed || targetExpansionState == .collapsed {
-                    move(to: .collapsed, animated: false)
-                }
-                delegate?.bottomSheetControllerCollapsedHeightInSafeAreaDidChange?(self)
+            if collapsedContentHeight != oldValue && isViewLoaded {
+                completeAnimationsIfNeeded(skipToEnd: true)
+                view.setNeedsLayout()
             }
         }
     }
 
     /// Indicates if the content should be hidden when the sheet is collapsed
     @objc open var shouldHideCollapsedContent: Bool = true {
         didSet {
-            if shouldHideCollapsedContent != oldValue {
-                updateExpandedContentAlpha()
+            if shouldHideCollapsedContent != oldValue && isViewLoaded {
```

Comment:
nit: use guard/return for these properties? (headerContentHeight, preferredExpandedContentHeight, collapsedContentHeight, shouldHideCollapsedContent, shouldAlwaysFillWidth,collapsedHeightInSafeArea)